### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by natalia-matash**
- [DEMO LINK](https://natalia-matash.github.io/layout_stars/)
- [TEST REPORT LINK](https://natalia-matash.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.stars--4 :nth-child(2),
+.stars--4 :nth-child(3),
+.stars--4 :nth-child(4),
+.stars--5 :nth-child(1),
+.stars--5 :nth-child(2),
+.stars--5 :nth-child(3),
+.stars--5 :nth-child(4),
+.stars--5 :nth-child(5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (nazarmatsevych) [inline src/style.css:44]:
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+.stars--0 :last-child,
+.stars--1 :last-child,
+.stars--2 :last-child,
+.stars--3 :last-child,
+.stars--4 :last-child,
+.stars--5 :last-child {
+  margin-right: 0;
+}
```
Comment (nazarmatsevych) [inline src/style.css:26]:
```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
```
Comment (nazarmatsevych) [inline src/style.css:11]: Now you can remove this margin

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (natalia-matash) [reply src/style.css]: Done

**Review by qoa11a — APPROVED** [review]

**Pull request closed without merging**